scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SchedulerImages.java

```diff
@@ -36,6 +36,7 @@
  * @author mschnoor
  *
  */
+@SuppressWarnings("squid:S1214")
```

Comment:
Could you please put comment why do you suppress this warning?

Reply:
@Mykhailenko I need to refactor this class, which I'm doing right away, but I need to do it in a separate PR, therefore I didn't want to put this personal reason in a comment. (TODO later)